frontend/src/components/LandingPage/Features/Features.test.js

```diff
@@ -0,0 +1,11 @@
+import Features from './Features'
+import { hasBeenRendered } from "../../../global";
+import { shallow } from "enzyme";
+
+const wrapper = shallow(<Features />);
+
+describe("Test Features Component", () => {
+    it("should render features component", () => {
+        hasBeenRendered(wrapper);
+    });
+});
```

Comment:
```
/**
 * Checks if the component inside the wrapper has the expected text.
 * 
 * @param {ShallowWrapper<any, Readonly<{}>, React.Component<{}, {}, any>>} wrapper Shallow wrapper.
 * @param {any} component Component.
 * @param {number} index Component index.
 * @param {string} expectedText Expected text.
 */
function hasText(wrapper, component, index, expectedText) {
    expect(wrapper.find(component).at(index).children().equals(expectedText)).toBeTruthy();
}
```

Reply:
Added the tests. Thank you for writing them!